meta-mbl-distro/classes/mbl-partitions.bbclass

```diff
@@ -204,7 +204,7 @@ MBL_PARTITION_NAMES += "WKS_BOOTLOADER_FS"
 #
 MBL_BOOT_MOUNT_POINT = "${MBL_BOOT_DIR}"
 MBL_BOOT_LABEL ?= "boot"
-MBL_BOOT_DEFAULT_SIZE_MiB ?= "48"
+MBL_BOOT_DEFAULT_SIZE_MiB ?= "128"
```

Comment:
Is there a reason we're increasing this almost 3x? Are we expecting the FIT image to grow that much larger? Is this within the typical size range for a boot partition on embedded systems? This is more out of curiosity than to gate the PR.

Reply:
It's mainly because the FIT image on this partition contains the initramfs, and being able to have a large initramfs could be quite useful to some people.

Reply:
> Is this within the typical size range for a boot partition on embedded systems?

I think this is on the larger side for embedded systems. If we add support for a targets with  smaller flash devices then we can still change the default sizes on a per-MACHINE basis.